packages/test-base/src/commonMain/kotlin/io/realm/kotlin/test/util/TypeDescriptor.kt

```diff
@@ -25,12 +25,21 @@ import io.realm.kotlin.types.RealmInstant
 import io.realm.kotlin.types.RealmObject
 import io.realm.kotlin.types.RealmUUID
 import org.mongodb.kbson.BsonObjectId
+import org.mongodb.kbson.Decimal128
 import kotlin.reflect.KClass
 import kotlin.reflect.KClassifier
 import kotlin.reflect.KMutableProperty1
 import kotlin.reflect.KType
 
-public object TypeDescriptor {
+object TypeDescriptor {
+    enum class AggregatorSupport {
+        MINMAX, SUM;
```

Comment:
It feels a bit weird that MINMAX covers two different ways. It would probably make more sense to split them?

Reply:
I don't think we would ever have a type that supports `min` but not `max`, but now I split it. It is also just in test infrastructure code so not that critical.